pakyow-support/lib/pakyow/support/defineable.rb

```diff
@@ -100,6 +122,12 @@ def stateful(name, object)
         def define(&block)
           instance_eval(&block)
         end
+
+        # Provide a default make method
+        #
+        def make(*args, &block)
+          new(*args, &block)
```

Comment:
Same as above, I was assuming that the object passed to stateful would be a `Defineable` object. If it was the case, then the block wouldn't need to be specifically `instance_eval`ed because `Defineable#initialize` already does that.